src/polyfills/string-starts-with.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
```

Comment:
👀

Reply:
You wouldn't believe when I started working on this file :)